Added more styling to form

helloimela opened the pull request:
- Added styling for disabled
- Added font-weight bold to the label
- Added styling for custom checkbox and custom radio buttons

**Solving issue**</br>

Fixes: #64

Commit:
Added more styling to form

- Added styling for disabled
- Added font-weight bold to the label
- Added styling for custom checkbox and custom radio buttons

Commit:
Merge branch 'master' into improvement/form_styling_2

Commit:
Move specific mixin to form css

- Moved form mixin inside form scss file so it is cleaner and more understandable

mmexvr commented on src/styles/components/_forms.scss:
```diff
@@ -81,6 +89,44 @@ select.form-control[multiple] {
 }
 
 // Form validation
+@mixin form-validation-state($state, $data) {
```
Should we add mixin with in the forms, since we don't do it with variables? 

I think that we should do the same solution for both mixin and variables,  e.g add only global variables in variable.scss and have variables only related with the element inside the element.scss

mmexvr reviewed (changes requested):
Found some sizing issue for IE with the checkmark 
![image](https://user-images.githubusercontent.com/35451568/66831607-5f819880-ef58-11e9-949d-8f2e5e9044e2.png)

Commit:
Fix styling for IE and precise pixel for radio button

- Fix check mark pixelated in IE
- Fix styling for checked radio box to be precisely in the middle of the circle
- Moved variable from global to component's level
